entities/Session.js

```diff
@@ -0,0 +1,69 @@
+module.exports =  function(_, pg, db){
+	
+	function makeid(){
+        var text = "";
+        var possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+
+        for( var i=0; i < 12; i++ )
+            text += possible.charAt(Math.floor(Math.random() * possible.length));
+        return text;
+    }
+
+    function createSession(data, callback){
```

Comment:
each custom 10 lines costs 1h (30$) per contributor to lern/maintain/update,  open source analogues costs nothing.